service/payment.go

```diff
@@ -109,7 +110,7 @@ func (service *PaymentService) CreatePaymentSession(w http.ResponseWriter, req *
 		return
 	}
 
-	// log.Trace("TODO log successful creation with details") // TODO
+	log.InfoR(req, "Successfully POST request for new payment resource: "+paymentResource.ID)
```

Comment:
It would be a good idea to log the http status as well . If this is a http request you would get a header "X-Request-Id" . This should be logged in as well  in context. I would use log.InfoC("request-id",).. This way you can track the entire request from start to end as it will have the same request-id.

Reply:
Status added.
`InfoR` already includes the context, and is the correct variant to use.